### ISSUE TITLE
Route handler returns json object with @Redirect

### ISSUE THREAD

**Issue by wowgs** [nestjs/nest#3305]
## Bug Report
Redirect works not properly

## Current behavior

If you use Redirect('', ) decorator with empty string as an argument, then return {url: _} returns a json object with 'url' as a property

## Input Code

```ts
@Redirect("", 302)
public async redirectGoogle() {
  return {
    url: "https://google.com"
  };
}
```

## Expected behavior

Redirecting to https://google.com

## Environment

<pre><code>
Nest version: 6.10.5

 
For Tooling issues:
- Node version: 12.12.0  
- Platform:  Mac

Others:

</code></pre>

**Comment by kamilmysliwiec** [comment]
cc @johnbiundo could you take a look? :)

**Comment by johnbiundo** [comment]
Fixed with #4085
